File: ANDROID/app/src/main/java/com/carapp/gobi/carapp/domain/Car.java
package com.carapp.gobi.carapp.domain;

import java.io.Serializable;


/**
 * Created by gobi on 11/9/2017.
 */
public class Car implements Serializable {

    private String id;

    private String chassisCode;

    private String model;

    private String make;

    private int cubicCapacity;

    private int modelYear;

    private volatile boolean isNew;

    public Car(String id, String chassisCode, String model, String make, int cubicCapacity, int modelYear) {
        this.id = id;
        this.chassisCode = chassisCode;
        this.model = model;
        this.make = make;
        this.cubicCapacity = cubicCapacity;
        this.modelYear = modelYear;
        this.isNew = false;
    }

    public Car() {
    }

    public String getChassisCode() {
        return chassisCode;
    }

    public void setChassisCode(String chassisCode) {
        this.chassisCode = chassisCode;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    public String getMake() {
        return make;
    }

    public void setMake(String make) {
        this.make = make;
    }

    public int getCubicCapacity() {
        return cubicCapacity;
    }

    public void setCubicCapacity(int cubicCapacity) {
        this.cubicCapacity = cubicCapacity;
    }

    public int getModelYear() {
        return modelYear;
    }

    public void setModelYear(int modelYear) {
        this.modelYear = modelYear;
    }


    @Override
    public String toString() {
        return "Car{" +
                "chassisCode='" + chassisCode + '\'' +
                ", model='" + model + '\'' +
                ", make='" + make + '\'' +
                ", cubicCapacity=" + cubicCapacity +
                ", modelYear=" + modelYear +
                '}';
    }

    public boolean isNew() {
        return isNew;
    }

    public void setNew(boolean aNew) {
        isNew = aNew;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}

File: REACTNATIVE/app/Car.js
export class Car {

    constructor(id, chassisCode, make, model, year, cubicCapacity){
        this.id = id;
        this.chassisCode = chassisCode;
        this.make = make;
        this.model = model;
        this.year = year;
        this.cubicCapacity = cubicCapacity;
        this.isNew = true;
    }

    getID(){
        return this.id;
    }

    setID(id){
        this.id = id;
    }

    getchassisCode() {
        return this.chassisCode;
    }

    setchassisCode(value) {
        this.chassisCode = value;
    }

    getmake() {
        return this.make;
    }

    setmake(value) {
        this.make = value;
    }

    getmodel() {
        return this.model;
    }

    setmodel(value) {
        this.model = value;
    }

    getyear() {
        return this.year;
    }

    setyear(value) {
        this.year = value;
    }

    getcubicCapacity() {
        return this.cubicCapacity;
    }

    setcubicCapacity(value) {
        this.cubicCapacity = value;
    }

    getisNew() {
        return this.isNew;
    }

    setIsNew(value) {
        this.isNew = value;
    }
}